scripts/alert_emails/template.html

```diff
@@ -209,7 +209,7 @@
                                             <p
                                                 style="Margin-top:0;Margin-bottom:0;font-style:normal;font-weight:normal;font-family:Arial,Helvetica,sans-serif;text-align:center;font-size:15px;line-height:21px">
                                                 <span style="line-height:inherit">
-                                                    <span style="line-height:inherit;color:#4f4f4f">As of March 2022, Covid Act Now follows the CDC's Community Level framework to provide a 3-color community level assessment for states and counties across the nation so citizens and government officials can better understand COVID status in their area. Learn more at <a href="https://covidactnow.org/about?utm_source=risk_alerts&utm_medium=email"
+                                                    <span style="line-height:inherit;color:#4f4f4f">As of March 2022, Covid Act Now follows the <a href="https://www.cdc.gov/coronavirus/2019-ncov/science/community-levels.html" style="line-height:inherit;color:#00bfea" target="_blank">CDC's Community Level framework</a> to provide a 3-color community level assessment for states and counties across the nation so citizens and government officials can better understand COVID status in their area. Learn more at <a href="https://covidactnow.org/about?utm_source=risk_alerts&utm_medium=email"
```

Comment:
I think we should change March to April to match https://covid-projections-git-can-82-covidactnow.vercel.app/covid-risk-levels-metrics (which I think originally said March but got updated to April at some point)